src/pages/docs/docs.config.json

```diff
@@ -29,6 +29,7 @@
     "/reference/tooling/piral": "/tooling/build-piral",
     "/reference/extensions/:id?": "/plugins/:id",
     "/reference/plugins/:id?": "/plugins/:id",
+    "/reference/codes/search": "/code/search",
```

Comment:
I don't think this is needed as /reference/codes/:id would match "search" as :id and therefore forward to /code/search properly.